tests/Doctrine/Tests/DBAL/Driver/Mysqli/MysqliConnectionTest.php

```diff
@@ -44,10 +44,10 @@ public function testDoesNotRequireQueryForServerVersion() : void
 
     public function testRestoresErrorHandlerOnException() : void
     {
-        $handler         = static function () : bool {
+        $handler        = static function () : bool {
```

Comment:
Would it make sense to add an empty line after the `$handler` declaration? It doesn't make much sense (even from the standard PoV) to keep the assignment of a multi-line expression aligned with the next one.